src/routes/dataInsights/+page.svelte

```diff
@@ -1,7 +1,22 @@
-<div class= "container justify-center mx-auto text-left p-24 md:text-center">
+<!--Headline-->
+<div class= "container justify-start mx-1 text-left p-24 md:text-left">
     <div class="relative z-[1] space-y-6 pt-32 pb-24">
-        <div class="font-bold text-4xl tracking-tight">
-            Data Insights Test <span class="text-primary-500">test</span>
+        <div class="font-bold text-6xl tracking-tight">
```

Comment:
For a size based font size: change `text-6xl` to `md:text-6xl text-xl` for example. This would change the font size to 6xl only if the width of the screen satisfies md (768px), if not it will use text-xl.